samples/betaFeatures.js

```diff
@@ -0,0 +1,586 @@
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(result => result.alternatives[0].transcript)
+        .join('\n');
+      const confidence = response.results
+        .map(result => result.alternatives[0].confidence)
+        .join(`\n`);
+      console.log(
+        ` Transcription: ${transcription} \n Confidence: ${confidence}`
+      );
+
+      console.log(` Word-Level-Confidence:`);
+      const words = response.results.map(result => result.alternatives[0]);
+      console.log(`WORDS: `);
+      words.forEach(a => console.log(JSON.stringify(a, null, 4)));
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END speechTranscribeWordLevelConfidence]
+}
+
+function speechTranscribeWordLevelConfidenceGCS(gcsUri) {
+  // [START speech_transcribe_word_level_confidence_gcs]
+
+  // Imports the Google Cloud client library
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  const config = {
+    encoding: `FLAC`,
+    sampleRateHertz: 16000,
+    languageCode: `en-US`,
+    enableWordTimeOffsets: true,
+    enableWordConfidence: true,
+  };
+
+  const audio = {
+    uri: gcsUri,
+  };
+
+  const request = {
+    config: config,
+    audio: audio,
+  };
+
+  client
+    .recognize(request)
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(result => result.alternatives[0].transcript)
+        .join('\n');
+      const confidence = response.results
+        .map(result => result.alternatives[0].confidence)
+        .join(`\n`);
+      console.log(
+        ` Transcription: ${transcription} \n Confidence: ${confidence}`
+      );
+
+      console.log(` Word-Level-Confidence:`);
+      const words = response.results.map(result => result.alternatives[0]);
+      console.log(`WORDS: `);
+      words.forEach(a => console.log(JSON.stringify(a, null, 4)));
+    })
+    .catch(err => {
+      console.error('ERROR:', err);
+    });
+  // [END speech_transcribe_word_level_confidence_gcs]
+}
+
+//audio file paths for Defaults
+const path = require(`path`);
+const monoFileName = `commercial_mono.wav`;
+const monoFilePath = path.join(
+  __dirname,
+  `../samples/resources/${monoFileName}`
+);
+// const defaultURI = 'gs://cloud-samples-tests/speech/commercial_mono.wav';
+const stereoFileName = `commercial_stereo.wav`;
+const stereoFilePath = path.join(
+  __dirname,
+  `../samples/resources/${stereoFileName}`
+);
+const multiLanguageFileName = `multi.wav`;
+const multiLanguageFile = path.join(
+  __dirname,
+  `../samples/resources/${multiLanguageFileName}`
+);
+//const multiLangURI = `gs://cloud-samples-tests/speech/multi.wav`;
+
+// const defaultStereoURI =
+//   'gs://cloud-samples-tests/speech/commercial_stereo.wav';
+const gnome = `/Users/cfrater/SPEECH3/nodejs-speech/samples/resources/Google_Gnome.wav`;
```

Comment:
I assume this is unintentional? If it's intentional, perhaps a relative path would be better?